### DISCUSSION TITLE
Does marked need to render HTML? Can it render Confluence wiki markup instead?

### DISCUSSION THREAD

**Discussion by jgornick** [markedjs/marked#2417] [Q&A] [answered]
As the title says, does marked _need_ to render HTML or can it render any sort of output? Any help would be greatly appreciated!

**Comment by jgornick** [comment, marked as answer]
I may have found my answer already: https://github.com/mikaelbr/marked-terminal

The answer seems to be **yes**!

**Comment by UziTech** [comment]
You can change the renderer to output whatever you want. By default it just outputs HTML.

[Renderer docs](https://marked.js.org/using_pro#renderer)

**Reply by jgornick** [reply]
Thank you @UziTech. A little more searching and I would have found my answer :)
